src/main/java/hardware/Swerve.java

```diff
@@ -105,17 +106,13 @@ public class Swerve {
   /**
    * Creates a new DriveSu1stem.
    */
+
+
   public Swerve() {
     resetEncoders();
     zeroHeading();
     setBrakeMode();
 
-    SmartDashboard.putNumberArray("Swerve/RealStates", new double[] {
-        0, 0, 0, 0, 0, 0, 0, 0
-    });
-    SmartDashboard.putNumberArray("Swerve/DesiredStates", desiredModuleStates);
-    SmartDashboard.putNumber("Swerve/RobotRotation", getPose().getRotation().getDegrees());
```

Comment:
We still want to log init values, this will make the field show up in advantage scope before we enable.

Reply:
(change these deleted lines to incorporate our new method of logging, rather than deleting them all together)